### DISCUSSION TITLE
Continuous aggregatess

### DISCUSSION THREAD

**Discussion by semirke** [timescale/timescaledb-toolkit#328] [General]
Hi,
Im struggling with continuous aggregate views. I got some set up and looked as if they work great. They were populated as they should've been.
But now weeks went buy and numerous data was accumulated and now when I run a simple "select * from values_fmin order by bucket desc limit 1;" timescale starts immense read writes and no result is returned in a sane amount of time.
Is update policy a must for a c.a.? our data literarily never changes, so there is no need for updating already calculated aggregates, and so I did not set up a refresh policy.
Thanks!

**Comment by fabriziomello** [comment]
@semirke I received an answer already in the slack channel: https://timescaledb.slack.com/archives/C4GT3N90X/p1642768154475000?thread_ts=1642767858.474700&cid=C4GT3N90X
